raft/doc.go

```diff
@@ -126,7 +126,7 @@ The total state machine handling loop will look something like this:
       n.Tick()
     case rd := <-s.Node.Ready():
       saveToStorage(rd.State, rd.Entries, rd.Snapshot)
-      send(rd.Messages)
+      sendInNextReady(rd.Messages)
```

Comment:
I don't think we want to replace the send() here. Here it actually means sending messages to other raft peers.

Reply:
Done. I missed that goland had replaced it in all the comments.